src/aurelia-flatpickr.js

```diff
@@ -77,7 +77,7 @@ export class AureliaFlatpickrCustomElement {
         for(let d = 0; d < modelDates.length; d++) {
             let modelDate = modelDates[d];
 
-            if (view.indexOf(modelDate) > -1) {
```

Comment:
@samueldjack I checked this and you are right, the method is buggy. I have added a commit that fixes the bugs in this method and adds testing in general and specifically, testing for this method. I don't really want to use .some() because there is no need to search for more than one match in that instance. Instead I am using findIndex(). Let me know you think.